common/src/IO/ObjParser.h

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright (C) 2010-2017 Kristian Duske
+
+ This file is part of TrenchBroom.
+
+ TrenchBroom is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ TrenchBroom is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with TrenchBroom. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef TRENCHBROOM_OBJPARSER_H
+#define TRENCHBROOM_OBJPARSER_H
+
+#include "Assets/Asset_Forward.h"
+#include "IO/EntityModelParser.h"
+#include "IO/IO_Forward.h"
+#include "IO/Parser.h"
+#include "IO/Tokenizer.h"
+
+#include <vecmath/forward.h>
+
+#include <string>
+#include <vector>
+
+namespace TrenchBroom {
+    class Logger;
+
+    namespace IO {
+        class ObjVertexRef {
```

Comment:
If everything is public, this might as well be a struct.

Reply:
Furthermore, this and `ObjFace` are not used outside of the parser, so you might just forward declare them here and put the declaration / definition inline into the cpp. You could also make them private dependent types of the parser itself to avoid having these names in the IO namespace entirely.

Reply:
5993f76 should resolve this.